core/test_network/src/lib.rs

```diff
@@ -64,6 +64,18 @@ pub fn get_key_pair() -> KeyPair {
     ).unwrap()
 }
 
+/// Get hex string represented genesis_config for ./genesis.json that is tied to hardcoded keys and chain_id
+pub fn get_genesis_encoded_signature() -> String {
```

Comment:
It's always possible to encode `GenesisSignature` when needed. It's better to operate with stronger types when possible. Thus,

```rust
pub fn get_genesis_signature() -> GenesisSignature {
```